src/app/app.component.html

```diff
@@ -21,80 +21,111 @@
-        <div *ngIf="product.type == 'dish'">
-          <h3>{{product.name}}</h3>
-          <img class="product_image" data-toggle="modal" attr.data-target="#productModal{{product.id}}"
-               src={{product.image}}/>
-          <br>
-          <div ngbDropdown class="d-inline-block">
-            <button class="btn btn-primary" id="dropdownBasic1" ngbDropdownToggle>Välj Protein</button>
-            <div ngbDropdownMenu aria-labelledby="dropdownBasic1">
-              <div *ngFor="let ingredient of product.ingredients">
-                <button class="dropdown-item">{{ingredient.name}}</button>
-              </div>
-            </div>
-          </div>
-          <div class="modal fade" id="productModal{{product.id}}" tabindex="-1" role="dialog"
-               aria-labelledby="productCloseup" aria-hidden="true">
-            <div class="modal-dialog" role="document">
-              <button type="button" class="close" aria-label="Close" data-dismiss="modal">
-                <span aria-hidden="true">&times;</span>
-              </button>
-              <img class="img-responsive large_product_image" src={{product.image}} alt="Product Image"/>
-              <div class="product-description">
-                {{product.description}}
-              </div>
-              <button type="button" class="btn btn-primary" data-dismiss="modal">Stäng</button>
-            </div>
+      <div class="modal fade" id="productModal{{product.id}}" tabindex="-1" role="dialog"
+           aria-labelledby="productCloseup" aria-hidden="true">
+        <div class="modal-dialog" role="document">
+          <button type="button" class="close" aria-label="Close" data-dismiss="modal">
+            <span aria-hidden="true">&times;</span>
+          </button>
+          <img class="img-responsive large_product_image" src={{product.image}} alt="Product Image"/>
+          <div class="product-description">
+            {{product.description}}
           </div>
+          <button type="button" class="btn btn-primary" data-dismiss="modal">Stäng</button>
         </div>
       </div>
     </div>
   </div>
-  <br>
-  <agm-map [latitude]="lat" [longitude]="lng" [zoom]=15>
-    <agm-marker [latitude]="lat" [longitude]="lng" [iconUrl]="'../assets/circle.png'"></agm-marker>
-  </agm-map>
+</div>
+<br>
+<agm-map [latitude]="lat" [longitude]="lng" [zoom]=15>
+  <agm-marker [latitude]="lat" [longitude]="lng" [iconUrl]="'../assets/circle.png'"></agm-marker>
+</agm-map>
 <footer class="footer">
-    <span class="footer-text">
-      <h5> Mån-Fre 11-19, Lör: 12-16.30 </h5>
-      <h5> Beställ mellan 11-18:30 </h5>
-      <h5> Telefon: 076-633 35 52 </h5>
-      <a target="_blank" href="https://goo.gl/maps/s523jHUZ3VL2"><h5> Sveavägen 13, 111 57 </h5></a>
-      <a target="_blank" href="https://www.facebook.com/kimchistan/"><i class="fa fa-facebook-square fa-4x" aria-hidden="true"></i></a>
-    </span>
+  <span class="footer-text">
+  <h5> Mån-Fre 11-19, Lör: 12-16.30 </h5>
```

Comment:
Why are we not displaying the opening times on sundays ?

Reply:
It's closed on Sundays :)

Reply:
then they need to fix they're facebook info ;)